general-knowledge/about-mycrypto/system-mapping.md

```diff
@@ -0,0 +1,333 @@
+---
+title: System Mapping
+description: A description of systems
+priority: 90
+date_published: '2019-05-08'
+date_modified: '2019-05-08'
+---
+
+## Overview
+
+This document is meant to allow hackers to skip the recon stage and get straight into trying to find vulnerabilities. It outlines the in-scope domains, the software behind them, and a brief description of their functions.
+
+Our responsible disclosure program is run on HackerOne - [https://hackerone.com/mycrypto](https://hackerone.com/mycrypto).
+
+![](../../assets/general-knowledge/about-mycrypto/system-mapping/separator.png)

```

Comment:
All images should have an `alt`-tag.

Reply:
Added in dcec9a54c33f3a0b3fa8715052e1a1f57534afb9